tests/views/test_social.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+import requests_mock
+import json
+from response_operations_ui import app
+
+with open('tests/test_data/case/case_details.json') as f:
+    case_details = json.load(f)
+
+CASE_SAMPLES_URL = f'{app.config["CASE_URL"]}/cases/sampleunitids'
+
+
+def postcode_url(postcode):
+    postcode = postcode.replace(' ', '+')
+    return f'{app.config["SAMPLE_URL"]}/samples/sampleunits?postcode={postcode}'
+
+
+def mock_sample_units(postcode):
+    with open('tests/test_data/sample/sample_units.json') as f:
+        sample_units = json.load(f)
+        sample_units[0]['sampleAttributes']['attributes']['Postcode'] = postcode
+    return sample_units
+
+
+class TestSocial(unittest.TestCase):
+
+    def setUp(self):
+        self.app = app.test_client()
```

Comment:
this will need to be updated when changes from master are on this branch:

```python
from response_operations_ui import create_app
...

     def setUp(self):
         self.app = create_app('TestingConfig')
         self.client = self.app.test_client()
...

        response = self.client.get(...
...
```